plugin/fzf.vim

```diff
@@ -648,13 +648,15 @@ function! s:split(dict)
   \ 'left':  ['vertical topleft', 'vertical resize', &columns],
   \ 'right': ['vertical botright', 'vertical resize', &columns] }
   let ppos = s:getpos()
+  let is_popup = v:false
   try
     if s:present(a:dict, 'window')
       if type(a:dict.window) == type({})
         if !has('nvim') && !has('patch-8.2.191')
           throw 'Vim 8.2.191 or later is required for pop-up window'
         end
         call s:popup(a:dict.window)
+        let is_popup = v:true
```

Comment:
We shouldn't use `v:false` or `v:true` in this function as they are not available on older versions of Vim.

Reply:
OK, I fixed.